examples/full.rs

```diff
@@ -1,46 +1,51 @@
 extern crate rustbreak;
-#[macro_use] extern crate lazy_static;
+#[macro_use] extern crate serde_derive;
 
-use rustbreak::{Database, Result as BreakResult};
+use rustbreak::Database;
 
-lazy_static! {
-    static ref DB: Database<String> = {
-        Database::open("music").unwrap()
-    };
+#[derive(Eq, PartialEq, Debug, Serialize, Deserialize, Clone)]
```

Comment:
Would it make sense to derive `Hash` here?

Reply:
Ah, this is an example -.-' its too late in the evening to do proper review I guess.